test/files/run/t4671.check

```diff
@@ -14,12 +14,16 @@ object Test extends ReplTest {
 object o { val file = sys.props("partest.cwd") + "/t4671.scala" }
 val s = scala.io.Source.fromFile(o.file)
 println(s.getLines.mkString("\n"))
+s.close()
```

Comment:
This issue was to demonstrate not exhausting the iterator, IIRC, so it keeps use of Source.